app/views/active_admin/kaminari/_next_page.html.erb

```diff
@@ -8,7 +8,7 @@
 -%>
 <% unless current_page.last? %>
   <%= link_to url, rel: 'next', remote: remote, class: "flex items-center justify-center px-2.5 py-3 h-8 leading-tight text-gray-500 dark:text-gray-400 hover:bg-gray-100 hover:text-gray-700 dark:hover:bg-gray-800 dark:hover:text-white rounded no-underline" do %>
-    <span class="sr-only"><%= t('views.pagination.next').html_safe %></span>
+    <span class="sr-only"><%= t('active_admin.pagination.next').html_safe %></span>
```

Comment:
not sure about this, since the default template is using this translation in a screen reader only span, wouldn't be better to remove `html_safe`?

Reply:
Yes, I think so. I left this as is because it's what Kaminari used but then it had to as it used HTML entities for arrows which we don't want. We just need the plain text. I'll leave it up to you to make that change if you want. This is all set to merge either way so thank you!

Reply:
I just rebased so the tests should now pass. Sorry about that. I was adding the changelog from 3.2.0 into default branch and forgot we run tests on the changelog but no more.